### PR TITLE
Fix small vot wording

### PR DESCRIPTION
**Pull request by PippaClarkGDS**
## Why

Wording not in-line with GDS style and error message needed to be consistent with other error messages. 

## What

Fixes wording to be in GDS style and amends the error message to be consistent. 

## Technical writer support

will ask Sabrina to review.

## How to review

Please check error message wording is correct. 

## Changelog

not needed. 

## Confirm

- [y] I have checked if any docs change here also requires updates to other repositories (ADRs / RFCs, README.md, Team Manual, elsewhere in these docs)
- [y] Where there is any overlap I have updated or opened a PR for corresponding changes

### CODE REVIEW

**Comment by PippaClarkGDS** [comment]
@ethanmills @philf999 I noticed some small errors re style and consistency on this PR merged last month https://github.com/govuk-one-login/tech-docs/pull/193/files. I've gone through and made some small changes - could you check all is correct (especially the error message as I had to infer some knowledge)?

File: source/integrate-with-integration-environment/authenticate-your-user.html.md.erb [hunk 1]
```diff
@@ -208,7 +208,7 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
...
-| `invalid_request - Request vtr not valid`| If you're logging in without second factor authentication (requesting medium confidence ('P2')), you must request a protection level of Cl.Cm or you will receive this error.
+| `invalid_request - Request vtr not valid`| You've logged in without two-factor authentication and requested a medium confidence level (`P2`). To fix, you must request a protection level of `Cl.Cm`.
```
Comment (ethanmills) [inline source/integrate-with-integration-environment/authenticate-your-user.html.md.erb:211]: This doesn't sound quite right. This is `You've requested single factor authentication and a medium confidence level` or similar

Comment (ethanmills) [reply source/integrate-with-integration-environment/authenticate-your-user.html.md.erb:211]: Past tense on "logged in" makes it incorrect. The P and C are requested at the same time, and the issue here is that the two values are incompatible

File: source/integrate-with-integration-environment/authenticate-your-user.html.md.erb [hunk 2]
```diff
@@ -353,7 +353,7 @@ You can use the following table to understand the ID token’s claims.
 | `exp`   | `exp` means ‘expiration time’. This is the expiration time for this token, which will be an integer timestamp representing the number of seconds since the [Unix Epoch][external.unix-epoch]. |
 | `iat`   | `iat` stands for ‘issued at’. This identifies the time at which GOV.UK One Login created the JWT. You can use this claim to understand the age of the JWT.This will appear as an integer timestamp representing the number of seconds since the [Unix Epoch][external.unix-epoch]. |
 | `sid`   | `sid` stands for ‘session identifier’. This uniquely identifies the user's journey within GOV.UK One Login. |
-| `vot`   | Check the `vot` matches the authentication protection level you requested in your authorise request. If you make an identity request, this vot will only contain the credential trust level, not the level of confidence.  |
+| `vot`   | `vot` stands for 'Vector of Trust'. Check the `vot` matches the authentication protection level you requested in your authorise request. If you make an identity request, the `vot` claim will only contain the credential trust level, not the level of confidence.  |
```
Comment (ethanmills) [inline source/integrate-with-integration-environment/authenticate-your-user.html.md.erb:356]: nitpick, but something like
```suggestion
| `vot`   | `vot` stands for 'Vector of Trust'. Check the `vot` matches the authentication protection level you requested in your authorise request. The `vot` claim will only contain the credential trust level, not the level of confidence, even if you make an identity request.  |
```
?

File: source/integrate-with-integration-environment/authenticate-your-user.html.md.erb (hunk 1 above)
Comment (PippaClarkGDS) [reply source/integrate-with-integration-environment/authenticate-your-user.html.md.erb:211]: cheers @ethanmills - good points. I was trying to make this error consistent with the other error codes appearing in the docs (i.e. stating what's happened and how to fix it). Would the following be an accurate way of describing this error: 

_You've requested single factor authentication and a medium confidence level (`P2`). Make sure you are requesting two-factor authentication and `P2`._

Happy to jump on a quick call if easier than threading.

File: source/integrate-with-integration-environment/authenticate-your-user.html.md.erb (hunk 2 above)
Comment (PippaClarkGDS) [reply source/integrate-with-integration-environment/authenticate-your-user.html.md.erb]: will amend the PR for this

**Comment by PippaClarkGDS** [comment]
@ethanmills thanks for the time yesterday, I've made that small change and this will now go to tech writer review.

**Comment by pauldougan** [comment]
given we have thumbs up from the second tech writer I am going to merge this

**Pull request merged**